pkg/services/accesscontrol/api/api.go

```diff
@@ -36,6 +43,18 @@ func (api *AccessControlAPI) RegisterAPIEndpoints() {
 	api.RouteRegister.Group("/api/access-control", func(rr routing.RouteRegister) {
 		rr.Get("/user/actions", middleware.ReqSignedIn, routing.Wrap(api.getUserActions))
 		rr.Get("/user/permissions", middleware.ReqSignedIn, routing.Wrap(api.getUserPermissions))
+
+		rr.Get("/policies", middleware.ReqOrgAdmin, func(c *contextmodel.ReqContext) response.Response {
```

Comment:
we should remove these changes before merging... but this is a helpful way to see what the permissions will resolve to